docs/en/pact-properties-api.md

```diff
@@ -635,6 +635,21 @@ List / string / object contains
 
 Supported in either invariants or properties.
 
+### enumerate {#FEnumerate}
```

Comment:
Just double-checking - we want this change to `enumerate` .md docs to get merged?

Reply:
This is fine. It's the docs for the enumerate _property_, not the native. Nothing to fear here.

Reply:
But this PR has nothing to do with enumerate. It should be taken out of this PR.

Reply:
This is outdated